docs/source/relayer.rst

```diff
@@ -0,0 +1,105 @@
+Cross-chain message relayer
+===========================
+
+| This component is responsible for handling the cross-chain contract state updates. 
+|
+| It is mainly used by the Agent software when resolving requests as a result of a running challenge game
+  or for sending fill invalidation messages (L1 resolutions). However, it can also be used manually by running
+  the CLI commands.
+| If you haven't read about the role of :ref:`l1-resolutions` in Beamer, make sure you get familiar 
+| with this concept first.
+|
+| Basically, every **request fill** action creates a **proof** on the request destination chain.
+  The agent utilizes these proofs in an optimistic way. If a challenge game ever occurs, or if a
+  fill was invalid, the agent has the chance to prove that a misbehavior occured and in the case of a challenge game,
+  get rewarded for it.  
+|
+| Since these **proofs** lay on a different chain than the chain where the requests originated from and where the challenge
```

Comment:
Is `lay` a typo? It looks like `lie` would be more fitting here. Or perhaps a rephrasing along the lines of `Since these proofs are created on a different...`

Reply:
I think lay is better as lie also means like some is cheating lying to somebody, but generally the best solution is just using ``Since these proofs are created on a different...`` like @istankovic suggested.